debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskConnectorConfig.java

```diff
@@ -0,0 +1,25 @@
+package io.debezium.connector.sqlserver;
```

Comment:
This file has a missing license header which fails the build (apart from other checkstyle issues).